extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/linkHandler/BandcampPlaylistLinkHandlerFactory.java

```diff
@@ -0,0 +1,31 @@
+// Created by Fynn Godau 2019, licensed GNU GPL version 3 or later
+
+package org.schabi.newpipe.extractor.services.bandcamp.linkHandler;
+
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+import org.schabi.newpipe.extractor.linkhandler.ListLinkHandlerFactory;
+
+import java.util.List;
+
+/**
+ * Just as with streams, the album ids are essentially useless for us.
+ */
+public class BandcampPlaylistLinkHandlerFactory extends ListLinkHandlerFactory {
+    @Override
+    public String getId(String url) throws ParsingException {
+        return getUrl(url);
+    }
+
+    @Override
+    public String getUrl(String url, List<String> contentFilter, String sortFilter) throws ParsingException {
+        if (url.endsWith("/"))
+            url = url.substring(0, url.length() - 1);
+        url = url.replace("http://", "https://").toLowerCase();
+        return url;
```

Comment:
There's no need to do this stuff…

Reply:
I wrote this because I thought that in order for album URLs to work as IDs, they would always have to look the same, so that asking for the ID of the same URL twice would have to always yield the same result

Reply:
YouTube channel IDs also aren't always the same. AFAIK we don't store IDs anywhere in NP, just URLs

Reply:
Okay, I removed "URL cleanup" with e08256ebc6ac71f3d1b3d94db1f81359e3e69cb9.